etc/env/l1-inits/.init.env

```diff
@@ -0,0 +1,23 @@
+CONTRACTS_CREATE2_FACTORY_ADDR=0x3c4CE2E9D8b03BbCd568E47465A483EE86893c49
```

Comment:
should this file be submitted?

Reply:
cc @kelemeno BTW do we really need this file committed? Currently whenever I do zk init it does changes to this file so I have to be careful not to include those to prevent bigger diff in the PRs to this branch

Reply:
it is not needed at least for CI to pass